### PR TITLE
Refactoring to use umbrella chart with Atomix in the default namespace

### PR DESCRIPTION
**Pull request by ray-milkey**

### CODE REVIEW

File: test/ha/suite.go [hunk 1]
```diff
@@ -15,60 +15,21 @@
...
-		Set("image.tag", "latest").
-		Set("replicaCount", "2").
-		Set("storage.controller", onostest.AtomixController(testName, onosComponentName)).
-		Install(true)
-	if err != nil {
-		return err
-	}
-
-	return nil
+	umbrella := charts.CreateUmbrellaRelease()
```
Comment (adibrastegarnia) [inline test/ha/suite.go:33]: For HA, we run two instance of onos-config but we are not ready yet to run two instance of onos-config for gnmi tests. Looks like we are using one function for both

Comment (kuujo) [reply test/ha/suite.go:33]: We should set the values on the returned release then rather than inside `CreateUmbrellaRelease()`.

Comment (kuujo) [reply test/ha/suite.go:33]: nvm I guess you already did that 🤷

File: benchmark/gnmi/suite.go [hunk 2]
```diff
@@ -37,52 +34,12 @@ type BenchmarkSuite struct {
...
-		Set("storage.controller", onostest.AtomixController(testName, onosComponentName)).
-		Set("image.tag", "latest").
-		Install(true)
-	if err != nil {
-		return err
-	}
-	return nil
+	umbrella := charts.CreateUmbrellaRelease().
+		Set("onos-topo.replicaCount", 2).
+		Set("onos-config.replicaCount", 2)
```
Comment (kuujo) [inline benchmark/gnmi/suite.go:41]: @adibrastegarnia this should be 1 replica, right?

File: test/ha/suite.go (hunk 1 above)
Comment (kuujo) [reply test/ha/suite.go:33]: nvm I guess we're already doing that, but this should set the replica count to `2`, and the gNMI test should leave it at `1` AIUI

File: benchmark/gnmi/suite.go (hunk 2 above)
Comment (ray-milkey) [reply benchmark/gnmi/suite.go:41]: The previous code set it to two. Not sure why.

Comment (kuujo) [reply benchmark/gnmi/suite.go:41]: I think that I was just being optimistic 😛  If the tests pass we should leave it there though.

**Review by adibrastegarnia — APPROVED** [review]

**Pull request merged**